### ISSUE TITLE
Error on Save

### ISSUE THREAD

**Issue by BigCasper** [twanvl/MagicSetEditor2#110]
Following on the "Assertion Error" thread: The problem seems to have evolved after update to 2.1.2. Great.
"[Window Title]
Error

[Main Instruction]
An internal error occured:

[Content]
referencing a nonexistant file
Please save your work (use 'save as' to so you don't overwrite things)
and restart Magic Set Editor.

You should leave a bug report on https://github.com/twanvl/MagicSetEditor2/issues/
Press Ctrl+C to copy this message to the clipboard.

Call stack:
[00] 00007FF629365567
[01] 00007FF629369784
[02] 00007FF629369C80
[03] 00007FF6291C7493
[04] 00007FF6291ABC8C
[05] 00007FF629226AA9
[06] 00007FF62936E3C2
[07] 00007FF62936CB01
[08] 00007FF6291F0A95
[09] 00007FF6291E9A77
[10] 00007FF6292C7122
[11] 00007FF6293053EC
[12] 00007FF6295512B4
[13] 00007FF62957FFB4
[14] 00007FF6295811E3
[15] 00007FF62957FEB8
[16] 00007FF6293E357E
[17] 00007FF62957FF0C
[18] 00007FF629580D3A
[19] 00007FF629439CB4
[20] 00007FF62948EC2B

[OK]"
The subsequent saves just spill out the program without error msgs.

**Comment by twanvl** [comment]
When does this error occur? Again when opening a file? If so, could you share the file that causes this crash? (if you don't want to do it publicly, you can email me at mail@twanvl.nl)

**Comment by BigCasper** [comment]
Already public:
https://drive.google.com/drive/folders/1XJ7DE1dI-kTTlSYa5so6Jig8GxzMJvpA?usp=sharing

**Comment by twanvl** [comment]
I don't have any issues when opening or saving the file you linked (I renamed it from `mse-set.bak` to `mse-set`). Are you doing anything else? And are you using recent versions of the templates (from https://magicseteditor.boards.net/thread/77/cajun-style-templates-symbol-exporter?page=1&scrollTo=354)?

**Comment by BigCasper** [comment]
Yes, I still had some problems after submission.
I looked inside my set with 7zip, and discovered that 1: there was an unrecognized masterpiece symbol file
2: several images were not written, or were empty instead.
3: the file I provided apparently wasn't the one I've discovered the above in. Excuse my dyslexia, i don't have that file anymore. If I run into any problem like that, I'll provide everything correctly next time.
Sorry for wasting time.
